recipes/jalview/build.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# compile jalview from source
+ant makedist
+
+# create target directory
+JALVIEWDIR=$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM
+mkdir -p $JALVIEWDIR
+
+# copy files to target
+cp -R dist/* $JALVIEWDIR/.
+
+# create wrapper script
+echo -e '#!/usr/bin/env bash\n\
+# Find original directory of bash script, resolving symlinks\
```

Comment:
Just for the sake of readability, wouldn't it be better it have this as separate file in the repo?

Reply:
jup, I decided the same last night..